tests/conftest.py

```diff
@@ -70,7 +70,9 @@
     DEFAULT_ABCI_PORT,
     DEFAULT_TENDERMINT_PORT,
     FlaskTendermintDockerImage,
-    TendermintDockerImage,
+)
+from aea_test_autonomy.fixture_helpers import (  # noqa: F401  # pylint: disable=unused-import
+    tendermint,
```

Comment:
Will this have any effect?

https://github.com/valory-xyz/open-autonomy/blob/8dea9b23fd6784891aeb41123cb8a2a6415f2d77/plugins/aea-test-autonomy/aea_test_autonomy/fixture_helpers.py#L61-L73

Reply:
pytest will register this a fixture since this is imported here